### PR TITLE
Use mobility-toolbox-js for Layers

### PR DESCRIPTION
**Pull request by edellucien**
# How to

Use mobility-toolbox-js lib to load layers
Use new react-spatial version

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [ ] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

**Comment by edellucien** [comment]
:warning: Needs MR to be merged:
https://github.com/geops/mobility-toolbox-js/pull/17
https://github.com/geops/react-spatial/pull/591

File: src/components/Map/Map.js [hunk 1]
```diff
@@ -216,16 +216,15 @@ class Map extends PureComponent {
...
-          layer.getVisible() &&
-          layer.isQueryable &&
-          (
-            layer.get('featureInfoEventTypes') || ['pointermove', 'singleclick']
-          ).includes(featureInfoEventType),
+    return layerService.getLayersAsFlatArray().filter((layer) => {
+      return (
+        layer.visible &&
+        layer.isQueryable &&
+        (layer.get('f') || ['pointermove', 'singleclick']).includes(
```
Comment (friedjoff) [inline src/components/Map/Map.js:223]: Wondering why `layer.get('featureInfoEventTypes')` has changed to `layer.get('f')`?

Comment (edellucien) [reply src/components/Map/Map.js:223]: Good catch! Sorry surely a Ctrl + f mistake :cold_sweat:

Comment (edellucien) [reply src/components/Map/Map.js]: Corrected :)

**Comment by friedjoff** [comment]
Also depends on https://github.com/geops/mobility-toolbox-js/pull/18

**Pull request merged**